static/js/message_events.js

```diff
@@ -154,6 +155,14 @@ exports.update_messages = function update_messages(events) {
             const compose_stream_name = compose_state.stream_name();
             const orig_topic = util.get_edit_event_orig_topic(event);
 
+            if (page_params.realm_allow_edit_history) {
+                edit_history_entry  = {
```

Comment:
Here is a whitespace anomaly -- two spaces before the `=`.

Reply:
Oh, my bad. I didn't notice this. I'll make the changes asap.